internal/service/ec2/vpc_flow_log_test.go

```diff
@@ -1136,3 +1167,83 @@ resource "aws_flow_log" "test" {
 }
 `, rName))
 }
+
+func testAccVPCFlowLogConfig_destinationTypeKinesisFirehose(rName string) string {
+	return acctest.ConfigCompose(testAccFlowLogConfigBase(rName), fmt.Sprintf(`
+
+resource "aws_flow_log" "test" {
+  log_destination      = aws_kinesis_firehose_delivery_stream.test.arn
+  log_destination_type = "kinesis-data-firehose"
+  traffic_type         = "ALL"
+  vpc_id               = aws_vpc.test.id
+}
+
+resource "aws_kinesis_firehose_delivery_stream" "test" {
+	name        = "kinesis_firehose_test"
```

Comment:
Can you double check that this and below are properly `terraform fmt` formatted?